doc/SpecifyingApplications/TestCases/Testing.md

```diff
@@ -0,0 +1,65 @@
+# Testing the applications
+
+Implementing testcases for IndividualPart of Service Layer and Oam Layer will suffice. As a set of common testcases (testcases for Basic part of Service Layer and Oam Layer) will be available in the [ApplicationPattern repository](https://github.com/openBackhaul/ApplicationPattern) which could be added to the actual collection.
+
+## Integrating test suites of Basic and Individual Parts
+
+The ApplicationPattern test cases can be downloaded or [cloned](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository) into the local system. [Import](https://learning.postman.com/docs/getting-started/importing-and-exporting-data/) the ApplicationPattern test suite into postman.
+
+- Copy the BasicPart from ServiceLayer of ApplicationPattern collection into ServiceLayer of application specific test suite
+- Copy the BasicPart from OamLayer of ApplicationPattern collection into OamLayer of application specific test suite.
+
+Now, The test suite can be tested against the mock server initially for basic corrections before testing against the real server.
+
+*Reference:* [*Mock Server*](../CreatingMock/CreatingMock.md)
+
+## Testing the server
+
```

Comment:
If possible can we add "customization to do in the load file before running it" ,for eg serverUrl, typeOfTesting , authorizationCode etc., to run the testcase.

Reply:
Updated as per comment